localization/replace/english/dlc/fp1/dlc_fp1_custom_loc_l_english.yml

```diff
@@ -0,0 +1,2 @@
+﻿l_english:
```

Comment:
there's already a file called like that directly in the replace/english folder, is this one really necessary?

Reply:
Must have missed that one